### ISSUE TITLE
MetaModelInc  module offline training  should call eval for validation predict.

### ISSUE THREAD

**Issue by caolongjie1101** [microsoft/qlib#1722]
## To Reproduce

1. create any module with  bn or dropout layer
2. run offline training 
3. inloop training missed  model.eval(), caused   batch normalization or dropout layer cannot work as predict model.

## Expected Behavior
prediction should not dropout any value or using dynamic batch mean and var for batch normalization.  The run_task function should be modified from :

def run_task(self, meta_input, phase):
        """ A single naive incremental learning task """
        .........
         with torch.no_grad():
            pred = self.framework(meta_input["X_test"].to(self.framework.device), None)
        return pred.detach().cpu().numpy()
to
def run_task(self, meta_input, phase):
        """ A single naive incremental learning task """
        .........
         with torch.no_grad():
            **_self.framework.eval()_**
            pred = self.framework(meta_input["X_test"].to(self.framework.device), None)
        return pred.detach().cpu().numpy()

**Comment by SarthakNikhal** [comment]
@you-n-g can I work on this issue?

**Comment by USTCKevinF** [comment]
We have checked out the snippet in qlib-main\qlib\contrib\meta\data_selection\model.py and we add self.tn.eval() in validation phase(line 73). Can you specify your question? Where did you find this problem
